diagrams/diagrams.md

```diff
@@ -137,32 +137,3 @@ sequenceDiagram title: Execute Quark Operation with Callback
     S -->> S: [14] Run Script
     S -->> T: [15] Erc20 Transfer "Repay" [sender=User Wallet]
 ```
-
-## Upgrade Quark Wallet
```

Comment:
Interesting to see this. I guess there was a point in time when we were using a fully upgradeable proxy for the Quark wallet.